lms/envs/aws.py

```diff
@@ -923,3 +923,5 @@
 
 # Courses categorization feature
 HOMEPAGE_COURSE_MAX = ENV_TOKENS.get('HOMEPAGE_COURSE_MAX', HOMEPAGE_COURSE_MAX)
+
+CATEGORIES = ENV_TOKENS.get("CATEGORIES", {})
```

Comment:
in case when in `common.py` is set `CATEGORIES`
`CATEGORIES = ENV_TOKENS.get("CATEGORIES", CATEGORIES)`